nutrition_labels/grant_data_processing.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import re
+# load data
+
+epmc_tags = pd.read_csv('data/raw/EPMC_relevant_tool_pubs_manual_edit.csv')
+rf_tags = pd.read_csv('data/raw/ResearchFish/research_fish_manual_edit.csv')
+grant_tags = pd.read_csv('data/raw/wellcome-grants-awarded-2005-2019_manual_edit.csv')
+grant_data = pd.read_csv('data/raw/wellcome-grants-awarded-2005-2019.csv')
```

Comment:
I take it you saved the original xlsx as a csv? do you mind uploaded your csv to the [sharepoint](https://wellcomecloud.sharepoint.com/sites/DataLabs/Machine%20Learning/Forms/AllItems.aspx?newTargetListUrl=%2Fsites%2FDataLabs%2FMachine%20Learning&viewpath=%2Fsites%2FDataLabs%2FMachine%20Learning%2FForms%2FAllItems%2Easpx&viewid=a99f51bc%2D20ec%2D4b35%2D9090%2D4eedc2450aec&id=%2Fsites%2FDataLabs%2FMachine%20Learning%2FNutrition%20Labels%2Fdata%2Fraw).
Just so we are working with the same file (just in case I change the formatting by saving it as a csv in a different way to how you did)

Reply:
can you also upload EPMC_relevant_tool_pubs_manual_edit.csv and research_fish_manual_edit.csv to the sharepoint too?

Reply:
oo and wellcome-grants-awarded-2005-2019_manual_edit.csv

Reply:
all done :)